src/views/token-swap/token-swap.view.jsx

```diff
@@ -10,45 +10,58 @@ import * as tokenSwapActions from '../../store/token-swap/token-swap.actions'
 import * as tokenSwapThunks from '../../store/token-swap/token-swap.thunks'
 import Container from '../shared/container/container.view'
 import { STEP_NAME } from '../../store/token-swap/token-swap.reducer'
-import Quotes from './components/quotes/quotes.view'
+import QuoteSelector from './components/quote-selector/quote-selector.view'
 import SwapForm from './components/swap-form/swap-form.view'
-import OfferSidenav from './components/offer-sidenav/offer-sidenav.view'
+import QuoteSidenav from './components/quote-sidenav/quote-sidenav.view'
+import { AmountBoxPosition } from './components/amount-box/amount-box.view'
 
 function TokenSwap ({
   currentStep,
-  steps,
+  quoteSidenav,
   preferredCurrency,
   accountsTask,
   quotesTask,
   fiatExchangeRatesTask,
-  selectedLpId,
   onLoadAccounts,
+  onLoadQuotes,
   onChangeHeader,
   onCleanup,
   onGoToQuotes,
-  onOpenOfferInfo,
-  onLoadQuotes,
-  onLoadingQuotes
+  onOpenQuoteSidenav,
+  onCloseQuoteSidenav
 }) {
   const classes = useTokenSwapStyles()
-
   const [amountFrom, setAmountFrom] = React.useState(BigNumber.from(0))
   const [amountTo, setAmountTo] = React.useState(BigNumber.from(0))
   const [selectedTokens, setSelectedTokens] = React.useState({})
-  const [isOfferSidenavOpen, setIsOfferSidenavOpen] = React.useState()
+  const [bestQuote, setBestQuote] = React.useState()
+  const [selectedQuote, setSelectedQuote] = React.useState(undefined)
+
+  React.useEffect(() => {
+    if (quotesTask.status === 'successful') {
+      const newBestQuote = quotesTask.data.reduce((acc, curr) => {
+        if (!acc) {
+          return curr
+        }
+
+        return BigNumber(curr.amountToToken).gt(BigNumber(acc.amountToToken))
```

Comment:
This line is still breaking the app should be something like this
`return BigNumber.from(curr.amountToToken).gt(BigNumber.from(acc.amountToToken))`